pkg/api/componentreadiness/regressiontracker.go

```diff
@@ -0,0 +1,363 @@
+package componentreadiness
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/google/uuid"
+	crtype "github.com/openshift/sippy/pkg/apis/api/componentreport"
+	"github.com/openshift/sippy/pkg/apis/cache"
+	v1 "github.com/openshift/sippy/pkg/apis/sippy/v1"
+	sippybigquery "github.com/openshift/sippy/pkg/bigquery"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/api/iterator"
+)
+
+const (
+	testRegressionsTable = "test_regressions_append"
```

Comment:
New table so I could clear out all the nullable fields due to evolving schema. Old table will be fully deleted, the data is bad by this point.